nomad/state/events_test.go

```diff
@@ -94,57 +94,6 @@ func TestEventFromChange_ACLTokenSecretID(t *testing.T) {
 	require.Empty(t, tokenEvent2.ACLToken.SecretID)
 }
 
-// TestEventFromChange_NodeSecretID ensures that a node's secret ID is not
```

Comment:
this check is provided in a new test: 
https://github.com/hashicorp/nomad/pull/10202/files#diff-4dd5ab69dbb20720939d5d0edc23108a3b6427943ff1b71f2fea90a79481c80dR168-R169

along with a sibling test for `/v1/node/:id` provided here:
https://github.com/hashicorp/nomad/pull/10202/files#diff-0acc6b10aaabc14cceeb1cbb9b9ded8935451d1f6b4514ee1806b5c9af1ff252R181